### ISSUE TITLE
Cannot download, network error

### ISSUE THREAD

**Issue by smlksmth** [Rob--W/crxviewer#90]
I am getting a network error, 401 on this extension:

https://chrome.google.com/webstore/detail/fut-autosniper/aljnpglojnmhfcdkdcgkcloolagikjnl

I paid for it but still cannot download the source.

**Comment by Rob--W** [comment]
Are you signed in to Chrome?

(Probably the same issue as #55. I can only investigate/fix this issue with some more details)

**Comment by smlksmth** [comment]
Yup! I can take a screencast shortly. Would seeing the console or any other
dev tools help see what’s happening?

**Comment by Rob--W** [comment]
The network tab of the devtools would help a lot. In Chrome, you can right-click on any network request item and choose the "Save all as HAR with content" option to export the request-response pair.

Note that the HAR file is NOT redacted, and it contains cookies. You could edit the content and manually redact the cookie values to make sure that the cookies don't cause issues. Or alternatively, sign out of Google and log in again to invalidate the cookies.

Send the HAR files by mail to me, so that its content doesn't become visible on Github, just in case.

**Comment by Rob--W** [comment]
Hi @smlksmth, do you have any information to share?

**Comment by smlksmth** [comment]
Sorry for the delay. I've just sent you an email.
